systemtests/src/test/java/io/enmasse/systemtest/clients/rhea/BasicClientTest.java

```diff
@@ -0,0 +1,33 @@
+package io.enmasse.systemtest.clients.rhea;
+
+import io.enmasse.systemtest.executor.client.rhea.RheaClientReceiver;
+import io.enmasse.systemtest.executor.client.rhea.RheaClientSender;
+import org.junit.Test;
+
+public class BasicClientTest extends io.enmasse.systemtest.clients.BasicClientTest {

```

Comment:
What about move those tests into the BasicClientTest in clients package?
Because we don't want to duplicate Test Classes for each client. We want do something like following
```
@Test
public void basicMessageTest() throws Exception {
       doBasicMessageTest(new RheaClientSender(), new RheaClientReceiver());
       doBasicMessageTest(new PythonClientSender(), new PythonClientReceiver());
       doBasicMessageTest(new JavaClientSender(), new JavaClientReceiver());
       ...etc
}
```

Reply:
For now we will keep this implementation.